blackbox/docs/sql/reference/create_function.txt

```diff
@@ -0,0 +1,65 @@
+.. highlight:: psql
+.. _ref-create-function:
+
+===============
+CREATE FUNCTION
+===============
+
+Create a new function.
+
+Synopsis
+========
+
+::
+
+    CREATE [OR REPLACE] FUNCTION [ schema_name. ] function_name
+        ( [ [arg_name] arg_type ] [, ...] ] )
+    RETURNS return_type
+    { LANGUAGE language_name
+      | CALLED ON NULL INPUT | RETURNS NULL ON NULL INPUT | STRICT
+      | AS 'definition'
+    }
+
+Description
+===========
+
+``CREATE FUNCTION`` creates a new user defined function.
+
+``CREATE OR REPLACE FUNCTION`` will either create a new function, or replace an
+existing function.
+
+Optionally, a function name can contain a schema name. If the schema is not
+defined, the function is created in the current schema.
+
+The signature of the function is defined by its schema, name, input arguments,
+and return type.
+
+You can overload functions by defining two functions of the same name, but have
+a different set of input arguments.
+
+Parameters
+==========
+
+:schema_name: The name of the schema in which the function is created.
+
+:function_name: The name of the function to create.
+
+:arg_name: The name given to an argument. Function arguments do not retain names,
+  but you can name them in your query for documentation purposes.
+
+:arg_type: The data type of the function's arguments, if any. See
```

Comment:
data type of a given argument ... 
the if any is confusing, i would just leave it out. i think you mean if the function has arguments, this is not clear imho